app/mailers/mailer.rb

```diff
@@ -3,11 +3,18 @@ class Mailer < ActionMailer::Base
   default_url_options[:protocol] = Gemcutter::PROTOCOL
 
   def email_reset(user)
-    @user = user
+    @user = User.find(user['id'])
     mail from: Clearance.configuration.mailer_sender,
-         to: user.email,
-         subject: I18n.t(:confirmation,
-           scope: [:clearance, :models, :clearance_mailer],
-           default: "Email address confirmation")
+         to: @user.email,
+         subject: I18n.t('mailer.confirmation_subject',
+           default: 'Please confirm your email address with RubyGems.org')
+  end
+
+  def email_confirmation(user)
+    @user = User.find(user['id'])
```

Comment:
I just scan all the uses of `email_confirmation` and on all of them it seems it use an instance of `User`. Why does it need to find the user again?